user_guide_src/source/extending/composer_packages.rst

```diff
@@ -0,0 +1,194 @@
+##########################
+Creating Composer Packages
+##########################
+
+You can make the :doc:`../general/modules` you create into Composer packages,
+or can create a Composer package for CodeIgniter 4.
+
+.. contents::
+    :local:
+    :depth: 2
+
+****************
+Folder Structure
+****************
+
+Here's a typical directory structure for a Composer package::
+
+    your-package-name/
```

Comment:
If it would be published to packagist, then it should have a `LICENSE`.

Reply:
Good point. Added.